nvidia-bert/ort_addon/ort_supplement/ort_supplement.py

```diff
@@ -43,99 +50,82 @@ def setup_onnxruntime_with_mpi(args):
 
         torch.distributed.init_process_group(backend='nccl')
 
-    from onnxruntime.capi._pybind_state import set_cuda_device_id 
-    set_cuda_device_id(args.local_rank)
+    # tell onnxruntime which device to use and seed its random generators
+    from onnxruntime import set_seed
+    set_seed(args.seed + args.world_rank)
 
-    from onnxruntime.capi._pybind_state import set_arena_extend_strategy, ArenaExtendStrategy
+    # tell onnxruntime to only allocate as much memory as strictly required
+    from onnxruntime import set_arena_extend_strategy, ArenaExtendStrategy
     set_arena_extend_strategy(ArenaExtendStrategy.kSameAsRequested)
```

Comment:
If this model really need this PR, let me know and we will discuss adding it to the public API

Reply:
That was added after the design was approved, so it wasnt part of changes we commited to
Keep using the internal namespace on an educational notebook might teach users what we dont want to encourage :)

Reply:
I agree. I will remove this is.